pymc/distributions/discrete.py

```diff
@@ -462,15 +462,16 @@ def rng_fn(cls, rng, q, beta, size):
 
 
 class DiscreteWeibull(Discrete):
-    R"""Discrete Weibull log-likelihood
+    """Discrete Weibull log-likelihood.
```

Comment:
The `R` is needed to tell python to treat the docstring as a "raw" string. Otherwise, things like `\b` or `\n` are considered single special characthers instead of being one backslash and one letter and that breaks the latex formulas.

Reply:
Got it.